modules/core/src/main/scala/org/tessellation/infrastructure/snapshot/processing/BlockAcceptanceLogic.scala

```diff
@@ -0,0 +1,132 @@
+package org.tessellation.infrastructure.snapshot.processing
```

Comment:
I'd say it's a `program`

```
snapshot/
  programs/
    BlockAcceptance
    BlockAcceptanceDriver
```
and I bet both can/should be in the domain 🤔

Reply:
Ok, so under that definition:

>Program is a part of business logic that combines multiple services

This class is definitely not a `program`.